rust/loqui_client/src/lib.rs

```diff
@@ -5,11 +5,11 @@ extern crate log;
 
 mod client;
 mod config;
-mod connection_handler;
+// XXX: This is required to be public due to a bug in the rust compiler.
```

Comment:
wat. Is this documented somewhere? If so maybe include a link?

Reply:
😬yeah, I added a link to the ticket right under.